Makefile

```diff
@@ -19,6 +19,8 @@
 #
 export GOPROXY?=https://proxy.golang.org/
 CONTROLLER_GEN_BIN_PATH := $(shell which controller-gen)
+PKGS_COVER ?= $(shell go list ./... | grep -v /vendor/ | grep -v /testdata/ | grep -v /test/ | sort | uniq)
```

Comment:
`sort -u` can replace `sort | uniq`.  Additionally, we can skip the `grep -v vendor` and `grep -v testdata`, since we don't have a vendor directory and `go list` skips testdata already.

Reply:
done